xslts/distributions-add-function.md

```diff
@@ -0,0 +1,72 @@
+## Description
+
+Ajoute un élément de type `gmd:function` à une distribution.
+
+
+## Paramètres
+
+`match-string` (requis)
```

Comment:
This section needs a little pimping :nail_care: 
I tried a markdown table but it's not translated to an html table.

Reply:
![image](https://github.com/user-attachments/assets/62e7f8c7-7a0b-43a9-8b31-177475b87e19)

Better with https://github.com/ecolabdata/ecospheres-isomorphe/pull/102 + https://github.com/ecolabdata/ecospheres-isomorphe/pull/103 :v: